src/shared/js/content.js

```diff
@@ -121,6 +144,7 @@ const addBlurCode = () => {
   $('body')
     .on('mouseenter', CSK_ENTRY_SELECTOR, handleCskEntryMouseenter)
     .on('mouseleave', CSK_ENTRY_SELECTOR, handleCskEntryMouseleave)
+    .on('mouseover', CSK_ENTRY_SELECTOR, handleCskEntryMouseenter)
```

Comment:
Does it makes sense to have this triggered on both mouseover and mouseenter?

Reply:
Maybe not. Without mouseover, I noticed the blur effect wouldn't trigger consistently when moving over items inside another component.

Reply:
ok. As long as it is not behaving strange, I guess it's fine